ambient.d.ts

```diff
@@ -283,8 +285,20 @@ declare module '@mojaloop/central-services-shared' {
 }
 
 declare module '@mojaloop/central-services-error-handling' {
+  interface APIErrorObject {
+    errorInformation: {
+      errorCode?: string;
+      errorDescription?: string;
+      extensionList?: {
+        extension: [{
```

Comment:
Is this the same as `Array<{key:string, value:string}>`?

Reply:
Yes